test/distributed/test_c10d_nccl.py

```diff
@@ -4173,14 +4187,16 @@ def test_short(self, timing_enabled):
 
         t = pickle.loads(torch._C._distributed_c10d._dump_nccl_trace())
         ver = t['version']
-        self.assertEqual(ver, "1.4")
+        self.assertEqual(ver, "1.5")
         pg_config = t['pg_config']
         self.assertEqual(len(pg_config), 1)
-        self.assertEqual(len(pg_config['0']), self.world_size)
+        self.assertEqual(len(pg_config['0']), 3)
```

Comment:
Is there a reason to use specific number instead of `self.world_size`? 
If there is a need, can you please comment what 3 represents so that a new comer could know how to modify this test?

Reply:
3 represents that there are three fields for each PG in pg_config dict, i.e. "name", "desc" and "ranks". I will add a comment for this.

Reply:
maybe just assert that 'name', 'desc', and 'ranks' are in the config? clearer that way